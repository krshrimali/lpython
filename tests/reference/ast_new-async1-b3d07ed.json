{
    "basename": "ast_new-async1-b3d07ed",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/async1.py",
    "infile_hash": "491b8acb051fb636a81ff7564388a9c4ade8f0bdf6ce1c5583bd1938",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast_new-async1-b3d07ed.stderr",
    "stderr_hash": "189bb553a7310dd9b86afb6a8bd78d3d28622a07ac6335e860c9efa9",
    "returncode": 1
}