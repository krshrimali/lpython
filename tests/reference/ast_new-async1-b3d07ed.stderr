syntax error: Token ':' is unexpected here
 --> tests/parser/async1.py:5:21
  |
5 |     async for x in y:
  |                     ^ 
